.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Problem
+Add the issue number if exists. Describe the problem otherwise.
```

Comment:
Can we make this text somehow very obvious that it needs to be removed by user? Maybe just make it _italic_?

Reply:
Commented out those lines using `<!--` and `-->` so that they won't show up in the result even if user didn't remove them.